From c97147d04df856b500049315b37e7b65586a604d Mon Sep 17 00:00:00 2001
From: Steven Barrett <steven@liquorix.net>
Date: Wed, 27 Jan 2021 17:39:54 -0600
Subject: [PATCH 6/8] ZEN: Increase max elevator name size

After renaming mq-deadline to mq-deadline-nodefault, setting the new
scheduler name would always fail.  Part of that reason was the module
alias that was fixed in "ZEN: Fix mq-deadline scheduler renaming", but
also the scheduler name was longer than ELV_NAME_MAX (by default, 15
usable characters).

Increase to 24 so we can support our longer IO scheduler names.
---
 include/linux/elevator.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/elevator.h b/include/linux/elevator.h
index b5abf740a..4bff11535 100644
--- a/include/linux/elevator.h
+++ b/include/linux/elevator.h
@@ -52,7 +52,7 @@ struct elevator_mq_ops {
 	void (*exit_icq)(struct io_cq *);
 };
 
-#define ELV_NAME_MAX	(16)
+#define ELV_NAME_MAX	(24)
 
 struct elv_fs_entry {
 	struct attribute attr;
-- 
2.31.1.621.g97eea85a0a

